subprojects/core/src/testFixtures/groovy/org/gradle/api/internal/file/TestFiles.java

```diff
@@ -96,4 +97,8 @@ public static JavaExecHandleFactory javaExecHandleFactory(File baseDir) {
     public static Factory<PatternSet> getPatternSetFactory() {
         return resolver().getPatternSetFactory();
     }
+
+    public static String systemSpecificAbsolutePath(String path) {
+        return OperatingSystem.current().isWindows()? "C:/" + path : "/" + path;
```

Comment:
Something like `new File("").absolutePath()` would probably be less likely to break on different machines.